[2015-11-04 08:59:32] local.ERROR: exception 'InvalidArgumentException' with message 'Command "migration" is not defined.

Did you mean one of these?
    migrate
    migrate:refresh
    migrate:status
    migrate:reset
    migrate:rollback
    migrate:install
    make:migration' in E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Application.php:549
Stack trace:
#0 E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Command\HelpCommand.php(76): Symfony\Component\Console\Application->find('migration')
#1 E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Command\Command.php(259): Symfony\Component\Console\Command\HelpCommand->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#2 E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Application.php(878): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#3 E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Application.php(195): Symfony\Component\Console\Application->doRunCommand(Object(Symfony\Component\Console\Command\HelpCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#4 E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Application.php(126): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#5 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(100): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#6 E:\xampp\htdocs\workspaces\blog\artisan(36): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#7 {main}  
[2015-11-04 09:12:53] local.ERROR: exception 'InvalidArgumentException' with message 'Command "migration" is not defined.

Did you mean one of these?
    migrate
    migrate:refresh
    migrate:status
    migrate:reset
    migrate:rollback
    migrate:install
    make:migration' in E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Application.php:549
Stack trace:
#0 E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Command\HelpCommand.php(76): Symfony\Component\Console\Application->find('migration')
#1 E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Command\Command.php(259): Symfony\Component\Console\Command\HelpCommand->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#2 E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Application.php(878): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#3 E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Application.php(195): Symfony\Component\Console\Application->doRunCommand(Object(Symfony\Component\Console\Command\HelpCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#4 E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Application.php(126): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#5 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(100): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#6 E:\xampp\htdocs\workspaces\blog\artisan(36): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#7 {main}  
[2015-11-04 09:15:22] local.ERROR: exception 'InvalidArgumentException' with message 'Command "make:migrate" is not defined.

Did you mean one of these?
    make:migration
    make:test
    make:request
    make:seeder
    make:middleware
    make:controller
    make:provider
    make:policy
    make:event
    make:console
    make:job
    make:listener
    make:model
    make:command' in E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Application.php:549
Stack trace:
#0 E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Application.php(192): Symfony\Component\Console\Application->find('make:migrate')
#1 E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Application.php(126): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#2 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(100): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#3 E:\xampp\htdocs\workspaces\blog\artisan(36): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#4 {main}  
[2015-11-04 09:15:59] local.ERROR: exception 'InvalidArgumentException' with message 'Command "migration" is not defined.

Did you mean one of these?
    migrate
    migrate:refresh
    migrate:status
    migrate:reset
    migrate:rollback
    migrate:install
    make:migration' in E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Application.php:549
Stack trace:
#0 E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Command\HelpCommand.php(76): Symfony\Component\Console\Application->find('migration')
#1 E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Command\Command.php(259): Symfony\Component\Console\Command\HelpCommand->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#2 E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Application.php(878): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#3 E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Application.php(195): Symfony\Component\Console\Application->doRunCommand(Object(Symfony\Component\Console\Command\HelpCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#4 E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Application.php(126): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#5 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(100): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#6 E:\xampp\htdocs\workspaces\blog\artisan(36): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#7 {main}  
[2015-11-04 09:18:25] local.ERROR: exception 'InvalidArgumentException' with message 'There are no commands defined in the "make:make" namespace.

Did you mean this?
    make' in E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Application.php:501
Stack trace:
#0 E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Application.php(535): Symfony\Component\Console\Application->findNamespace('make:make')
#1 E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Application.php(192): Symfony\Component\Console\Application->find('make:make:migra...')
#2 E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Application.php(126): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#3 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(100): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#4 E:\xampp\htdocs\workspaces\blog\artisan(36): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#5 {main}  
[2015-11-04 09:18:42] local.ERROR: exception 'RuntimeException' with message 'Not enough arguments.' in E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Input\Input.php:77
Stack trace:
#0 E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Command\Command.php(254): Symfony\Component\Console\Input\Input->validate()
#1 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Console\Command.php(136): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#2 E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Application.php(878): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#3 E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Application.php(195): Symfony\Component\Console\Application->doRunCommand(Object(Illuminate\Database\Console\Migrations\MigrateMakeCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#4 E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Application.php(126): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#5 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(100): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#6 E:\xampp\htdocs\workspaces\blog\artisan(36): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#7 {main}  
[2015-11-04 09:20:47] local.ERROR: exception 'InvalidArgumentException' with message 'Command "migrate:make" is not defined.

Did you mean one of these?
    migrate:reset
    migrate:refresh
    migrate:status
    migrate:rollback
    migrate:install
    migrate' in E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Application.php:549
Stack trace:
#0 E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Application.php(192): Symfony\Component\Console\Application->find('migrate:make')
#1 E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Application.php(126): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#2 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(100): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#3 E:\xampp\htdocs\workspaces\blog\artisan(36): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#4 {main}  
[2015-11-04 09:21:11] local.ERROR: exception 'RuntimeException' with message 'Not enough arguments.' in E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Input\Input.php:77
Stack trace:
#0 E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Command\Command.php(254): Symfony\Component\Console\Input\Input->validate()
#1 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Console\Command.php(136): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#2 E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Application.php(878): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#3 E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Application.php(195): Symfony\Component\Console\Application->doRunCommand(Object(Illuminate\Database\Console\Migrations\MigrateMakeCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#4 E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Application.php(126): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#5 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(100): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#6 E:\xampp\htdocs\workspaces\blog\artisan(36): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#7 {main}  
[2015-11-04 09:41:28] local.ERROR: exception 'PDOException' with message 'SQLSTATE[HY000] [1045] Access denied for user 'homestead'@'localhost' (using password: YES)' in E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Database\Connectors\Connector.php:55
Stack trace:
#0 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Database\Connectors\Connector.php(55): PDO->__construct('mysql:host=loca...', 'homestead', 'secret', Array)
#1 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Database\Connectors\MySqlConnector.php(22): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=loca...', Array, Array)
#2 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Database\Connectors\ConnectionFactory.php(60): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#3 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Database\Connectors\ConnectionFactory.php(49): Illuminate\Database\Connectors\ConnectionFactory->createSingleConnection(Array)
#4 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Database\DatabaseManager.php(175): Illuminate\Database\Connectors\ConnectionFactory->make(Array, 'mysql')
#5 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Database\DatabaseManager.php(67): Illuminate\Database\DatabaseManager->makeConnection('mysql')
#6 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Database\Migrations\DatabaseMigrationRepository.php(171): Illuminate\Database\DatabaseManager->connection(NULL)
#7 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Database\Migrations\DatabaseMigrationRepository.php(139): Illuminate\Database\Migrations\DatabaseMigrationRepository->getConnection()
#8 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Database\Migrations\Migrator.php(393): Illuminate\Database\Migrations\DatabaseMigrationRepository->repositoryExists()
#9 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Database\Console\Migrations\MigrateCommand.php(100): Illuminate\Database\Migrations\Migrator->repositoryExists()
#10 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Database\Console\Migrations\MigrateCommand.php(58): Illuminate\Database\Console\Migrations\MigrateCommand->prepareDatabase()
#11 [internal function]: Illuminate\Database\Console\Migrations\MigrateCommand->fire()
#12 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Container\Container.php(503): call_user_func_array(Array, Array)
#13 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Console\Command.php(150): Illuminate\Container\Container->call(Array)
#14 E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Command\Command.php(259): Illuminate\Console\Command->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#15 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Console\Command.php(136): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#16 E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Application.php(878): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#17 E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Application.php(195): Symfony\Component\Console\Application->doRunCommand(Object(Illuminate\Database\Console\Migrations\MigrateCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#18 E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Application.php(126): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#19 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(100): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#20 E:\xampp\htdocs\workspaces\blog\artisan(36): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#21 {main}  
[2015-11-04 09:42:47] local.ERROR: exception 'PDOException' with message 'SQLSTATE[42000]: Syntax error or access violation: 1075 Incorrect table definition; there can be only one auto column and it must be defined as a key' in E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Database\Connection.php:390
Stack trace:
#0 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Database\Connection.php(390): PDOStatement->execute(Array)
#1 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Database\Connection.php(644): Illuminate\Database\Connection->Illuminate\Database\{closure}(Object(Illuminate\Database\MySqlConnection), 'create table `l...', Array)
#2 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Database\Connection.php(611): Illuminate\Database\Connection->runQueryCallback('create table `l...', Array, Object(Closure))
#3 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Database\Connection.php(391): Illuminate\Database\Connection->run('create table `l...', Array, Object(Closure))
#4 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Database\Schema\Blueprint.php(76): Illuminate\Database\Connection->statement('create table `l...')
#5 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Database\Schema\Builder.php(191): Illuminate\Database\Schema\Blueprint->build(Object(Illuminate\Database\MySqlConnection), Object(Illuminate\Database\Schema\Grammars\MySqlGrammar))
#6 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Database\Schema\Builder.php(134): Illuminate\Database\Schema\Builder->build(Object(Illuminate\Database\Schema\Blueprint))
#7 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(213): Illuminate\Database\Schema\Builder->create('users', Object(Closure))
#8 E:\xampp\htdocs\workspaces\blog\database\migrations\2014_10_12_000000_create_users_table.php(27): Illuminate\Support\Facades\Facade::__callStatic('create', Array)
#9 E:\xampp\htdocs\workspaces\blog\database\migrations\2014_10_12_000000_create_users_table.php(27): Illuminate\Support\Facades\Schema::create('users', Object(Closure))
#10 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Database\Migrations\Migrator.php(135): CreateUsersTable->up()
#11 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Database\Migrations\Migrator.php(112): Illuminate\Database\Migrations\Migrator->runUp('2014_10_12_0000...', 1, false)
#12 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Database\Migrations\Migrator.php(85): Illuminate\Database\Migrations\Migrator->runMigrationList(Array, false)
#13 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Database\Console\Migrations\MigrateCommand.php(74): Illuminate\Database\Migrations\Migrator->run('E:\\xampp\\htdocs...', false)
#14 [internal function]: Illuminate\Database\Console\Migrations\MigrateCommand->fire()
#15 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Container\Container.php(503): call_user_func_array(Array, Array)
#16 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Console\Command.php(150): Illuminate\Container\Container->call(Array)
#17 E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Command\Command.php(259): Illuminate\Console\Command->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#18 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Console\Command.php(136): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#19 E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Application.php(878): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#20 E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Application.php(195): Symfony\Component\Console\Application->doRunCommand(Object(Illuminate\Database\Console\Migrations\MigrateCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#21 E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Application.php(126): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#22 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(100): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#23 E:\xampp\htdocs\workspaces\blog\artisan(36): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#24 {main}

Next exception 'Illuminate\Database\QueryException' with message 'SQLSTATE[42000]: Syntax error or access violation: 1075 Incorrect table definition; there can be only one auto column and it must be defined as a key (SQL: create table `ld_users` (`id` int unsigned not null auto_increment primary key, `username` varchar(50) not null, `password` varchar(60) not null, `phone` int not null auto_increment primary key, `email` varchar(255) not null, `gender` tinyint not null auto_increment primary key, `head` varchar(255) not null, `type` tinyint not null auto_increment primary key, `status` tinyint not null auto_increment primary key, `remember_token` varchar(100) null, `created_at` timestamp default 0 not null, `updated_at` timestamp default 0 not null) default character set utf8 collate utf8_general_ci)' in E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Database\Connection.php:651
Stack trace:
#0 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Database\Connection.php(611): Illuminate\Database\Connection->runQueryCallback('create table `l...', Array, Object(Closure))
#1 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Database\Connection.php(391): Illuminate\Database\Connection->run('create table `l...', Array, Object(Closure))
#2 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Database\Schema\Blueprint.php(76): Illuminate\Database\Connection->statement('create table `l...')
#3 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Database\Schema\Builder.php(191): Illuminate\Database\Schema\Blueprint->build(Object(Illuminate\Database\MySqlConnection), Object(Illuminate\Database\Schema\Grammars\MySqlGrammar))
#4 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Database\Schema\Builder.php(134): Illuminate\Database\Schema\Builder->build(Object(Illuminate\Database\Schema\Blueprint))
#5 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(213): Illuminate\Database\Schema\Builder->create('users', Object(Closure))
#6 E:\xampp\htdocs\workspaces\blog\database\migrations\2014_10_12_000000_create_users_table.php(27): Illuminate\Support\Facades\Facade::__callStatic('create', Array)
#7 E:\xampp\htdocs\workspaces\blog\database\migrations\2014_10_12_000000_create_users_table.php(27): Illuminate\Support\Facades\Schema::create('users', Object(Closure))
#8 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Database\Migrations\Migrator.php(135): CreateUsersTable->up()
#9 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Database\Migrations\Migrator.php(112): Illuminate\Database\Migrations\Migrator->runUp('2014_10_12_0000...', 1, false)
#10 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Database\Migrations\Migrator.php(85): Illuminate\Database\Migrations\Migrator->runMigrationList(Array, false)
#11 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Database\Console\Migrations\MigrateCommand.php(74): Illuminate\Database\Migrations\Migrator->run('E:\\xampp\\htdocs...', false)
#12 [internal function]: Illuminate\Database\Console\Migrations\MigrateCommand->fire()
#13 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Container\Container.php(503): call_user_func_array(Array, Array)
#14 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Console\Command.php(150): Illuminate\Container\Container->call(Array)
#15 E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Command\Command.php(259): Illuminate\Console\Command->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#16 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Console\Command.php(136): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#17 E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Application.php(878): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#18 E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Application.php(195): Symfony\Component\Console\Application->doRunCommand(Object(Illuminate\Database\Console\Migrations\MigrateCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#19 E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Application.php(126): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#20 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(100): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#21 E:\xampp\htdocs\workspaces\blog\artisan(36): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#22 {main}  
[2015-11-05 02:38:32] local.ERROR: exception 'InvalidArgumentException' with message 'Command "server" is not defined.

Did you mean this?
    serve' in E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Application.php:549
Stack trace:
#0 E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Application.php(192): Symfony\Component\Console\Application->find('server')
#1 E:\xampp\htdocs\workspaces\blog\vendor\symfony\console\Application.php(126): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#2 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(100): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#3 E:\xampp\htdocs\workspaces\blog\artisan(36): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#4 {main}  
[2015-11-05 03:09:27] local.ERROR: exception 'UnexpectedValueException' with message 'Invalid route action: [App\Http\Controllers\AuthController]' in E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\Route.php:637
Stack trace:
#0 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\Route.php(107): Illuminate\Routing\Route->parseAction(Array)
#1 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\Router.php(518): Illuminate\Routing\Route->__construct(Array, 'auth/login', Array)
#2 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\Router.php(494): Illuminate\Routing\Router->newRoute(Array, 'auth/login', Array)
#3 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\Router.php(472): Illuminate\Routing\Router->createRoute(Array, 'auth/login', 'AuthController')
#4 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\Router.php(132): Illuminate\Routing\Router->addRoute(Array, 'auth/login', 'AuthController')
#5 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(213): Illuminate\Routing\Router->get('auth/login', 'AuthController')
#6 E:\xampp\htdocs\workspaces\blog\app\Http\routes.php(13): Illuminate\Support\Facades\Facade::__callStatic('get', Array)
#7 E:\xampp\htdocs\workspaces\blog\app\Http\routes.php(13): Illuminate\Support\Facades\Route::get('auth/login', 'AuthController')
#8 E:\xampp\htdocs\workspaces\blog\app\Providers\RouteServiceProvider.php(41): require('E:\\xampp\\htdocs...')
#9 [internal function]: App\Providers\RouteServiceProvider->App\Providers\{closure}(Object(Illuminate\Routing\Router))
#10 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\Router.php(350): call_user_func(Object(Closure), Object(Illuminate\Routing\Router))
#11 E:\xampp\htdocs\workspaces\blog\app\Providers\RouteServiceProvider.php(42): Illuminate\Routing\Router->group(Array, Object(Closure))
#12 [internal function]: App\Providers\RouteServiceProvider->map(Object(Illuminate\Routing\Router))
#13 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Container\Container.php(503): call_user_func_array(Array, Array)
#14 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Support\Providers\RouteServiceProvider.php(72): Illuminate\Container\Container->call(Array)
#15 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Support\Providers\RouteServiceProvider.php(30): Illuminate\Foundation\Support\Providers\RouteServiceProvider->loadRoutes()
#16 E:\xampp\htdocs\workspaces\blog\app\Providers\RouteServiceProvider.php(29): Illuminate\Foundation\Support\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#17 [internal function]: App\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#18 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Container\Container.php(503): call_user_func_array(Array, Array)
#19 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(734): Illuminate\Container\Container->call(Array)
#20 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(717): Illuminate\Foundation\Application->bootProvider(Object(App\Providers\RouteServiceProvider))
#21 [internal function]: Illuminate\Foundation\Application->Illuminate\Foundation\{closure}(Object(App\Providers\RouteServiceProvider), 17)
#22 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(718): array_walk(Array, Object(Closure))
#23 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\BootProviders.php(17): Illuminate\Foundation\Application->boot()
#24 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(203): Illuminate\Foundation\Bootstrap\BootProviders->bootstrap(Object(Illuminate\Foundation\Application))
#25 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(222): Illuminate\Foundation\Application->bootstrapWith(Array)
#26 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(117): Illuminate\Foundation\Http\Kernel->bootstrap()
#27 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(87): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#28 E:\xampp\htdocs\workspaces\blog\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#29 E:\xampp\htdocs\workspaces\blog\server.php(21): require_once('E:\\xampp\\htdocs...')
#30 {main}  
[2015-11-05 03:13:38] local.ERROR: exception 'UnexpectedValueException' with message 'Invalid route action: [App\Http\Controllers\AuthController]' in E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\Route.php:637
Stack trace:
#0 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\Route.php(107): Illuminate\Routing\Route->parseAction(Array)
#1 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\Router.php(518): Illuminate\Routing\Route->__construct(Array, 'auth/login', Array)
#2 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\Router.php(494): Illuminate\Routing\Router->newRoute(Array, 'auth/login', Array)
#3 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\Router.php(472): Illuminate\Routing\Router->createRoute(Array, 'auth/login', 'AuthController')
#4 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\Router.php(132): Illuminate\Routing\Router->addRoute(Array, 'auth/login', 'AuthController')
#5 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(213): Illuminate\Routing\Router->get('auth/login', 'AuthController')
#6 E:\xampp\htdocs\workspaces\blog\app\Http\routes.php(13): Illuminate\Support\Facades\Facade::__callStatic('get', Array)
#7 E:\xampp\htdocs\workspaces\blog\app\Http\routes.php(13): Illuminate\Support\Facades\Route::get('auth/login', 'AuthController')
#8 E:\xampp\htdocs\workspaces\blog\app\Providers\RouteServiceProvider.php(41): require('E:\\xampp\\htdocs...')
#9 [internal function]: App\Providers\RouteServiceProvider->App\Providers\{closure}(Object(Illuminate\Routing\Router))
#10 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\Router.php(350): call_user_func(Object(Closure), Object(Illuminate\Routing\Router))
#11 E:\xampp\htdocs\workspaces\blog\app\Providers\RouteServiceProvider.php(42): Illuminate\Routing\Router->group(Array, Object(Closure))
#12 [internal function]: App\Providers\RouteServiceProvider->map(Object(Illuminate\Routing\Router))
#13 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Container\Container.php(503): call_user_func_array(Array, Array)
#14 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Support\Providers\RouteServiceProvider.php(72): Illuminate\Container\Container->call(Array)
#15 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Support\Providers\RouteServiceProvider.php(30): Illuminate\Foundation\Support\Providers\RouteServiceProvider->loadRoutes()
#16 E:\xampp\htdocs\workspaces\blog\app\Providers\RouteServiceProvider.php(29): Illuminate\Foundation\Support\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#17 [internal function]: App\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#18 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Container\Container.php(503): call_user_func_array(Array, Array)
#19 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(734): Illuminate\Container\Container->call(Array)
#20 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(717): Illuminate\Foundation\Application->bootProvider(Object(App\Providers\RouteServiceProvider))
#21 [internal function]: Illuminate\Foundation\Application->Illuminate\Foundation\{closure}(Object(App\Providers\RouteServiceProvider), 17)
#22 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(718): array_walk(Array, Object(Closure))
#23 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\BootProviders.php(17): Illuminate\Foundation\Application->boot()
#24 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(203): Illuminate\Foundation\Bootstrap\BootProviders->bootstrap(Object(Illuminate\Foundation\Application))
#25 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(222): Illuminate\Foundation\Application->bootstrapWith(Array)
#26 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(117): Illuminate\Foundation\Http\Kernel->bootstrap()
#27 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(87): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#28 E:\xampp\htdocs\workspaces\blog\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#29 E:\xampp\htdocs\workspaces\blog\server.php(21): require_once('E:\\xampp\\htdocs...')
#30 {main}  
[2015-11-05 03:14:02] local.ERROR: exception 'ReflectionException' with message 'Class App\Http\Controllers\AuthController does not exist' in E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Container\Container.php:737
Stack trace:
#0 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Container\Container.php(737): ReflectionClass->__construct('App\\Http\\Contro...')
#1 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Container\Container.php(627): Illuminate\Container\Container->build('App\\Http\\Contro...', Array)
#2 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(674): Illuminate\Container\Container->make('App\\Http\\Contro...', Array)
#3 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(85): Illuminate\Foundation\Application->make('App\\Http\\Contro...')
#4 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(57): Illuminate\Routing\ControllerDispatcher->makeController('App\\Http\\Contro...')
#5 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\Route.php(201): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'App\\Http\\Contro...', 'login')
#6 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\Route.php(134): Illuminate\Routing\Route->runWithCustomDispatcher(Object(Illuminate\Http\Request))
#7 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\Router.php(704): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#8 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#9 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#11 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#12 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\Router.php(706): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#13 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\Router.php(671): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#14 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\Router.php(631): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#15 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(236): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#16 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#17 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#18 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\VerifyCsrfToken.php(50): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#19 [internal function]: Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#20 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#21 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\View\Middleware\ShareErrorsFromSession.php(49): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#22 [internal function]: Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#23 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#24 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Session\Middleware\StartSession.php(62): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#25 [internal function]: Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#26 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#27 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse.php(37): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#28 [internal function]: Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#29 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#30 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\EncryptCookies.php(59): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#31 [internal function]: Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#32 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#33 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(42): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#34 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#35 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#36 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#37 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#38 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(122): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#39 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(87): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#40 E:\xampp\htdocs\workspaces\blog\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#41 E:\xampp\htdocs\workspaces\blog\server.php(21): require_once('E:\\xampp\\htdocs...')
#42 {main}  
[2015-11-05 03:14:43] local.ERROR: exception 'ReflectionException' with message 'Class App\Http\Controllers\AuthController does not exist' in E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Container\Container.php:737
Stack trace:
#0 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Container\Container.php(737): ReflectionClass->__construct('App\\Http\\Contro...')
#1 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Container\Container.php(627): Illuminate\Container\Container->build('App\\Http\\Contro...', Array)
#2 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(674): Illuminate\Container\Container->make('App\\Http\\Contro...', Array)
#3 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(85): Illuminate\Foundation\Application->make('App\\Http\\Contro...')
#4 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(57): Illuminate\Routing\ControllerDispatcher->makeController('App\\Http\\Contro...')
#5 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\Route.php(201): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'App\\Http\\Contro...', 'login')
#6 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\Route.php(134): Illuminate\Routing\Route->runWithCustomDispatcher(Object(Illuminate\Http\Request))
#7 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\Router.php(704): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#8 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#9 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#11 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#12 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\Router.php(706): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#13 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\Router.php(671): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#14 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\Router.php(631): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#15 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(236): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#16 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#17 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#18 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\VerifyCsrfToken.php(50): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#19 [internal function]: Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#20 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#21 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\View\Middleware\ShareErrorsFromSession.php(49): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#22 [internal function]: Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#23 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#24 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Session\Middleware\StartSession.php(62): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#25 [internal function]: Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#26 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#27 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse.php(37): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#28 [internal function]: Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#29 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#30 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\EncryptCookies.php(59): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#31 [internal function]: Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#32 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#33 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(42): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#34 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#35 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#36 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#37 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#38 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(122): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#39 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(87): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#40 E:\xampp\htdocs\workspaces\blog\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#41 E:\xampp\htdocs\workspaces\blog\server.php(21): require_once('E:\\xampp\\htdocs...')
#42 {main}  
[2015-11-05 03:15:07] local.ERROR: exception 'ReflectionException' with message 'Class App\Http\Controllers\AuthController does not exist' in E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Container\Container.php:737
Stack trace:
#0 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Container\Container.php(737): ReflectionClass->__construct('App\\Http\\Contro...')
#1 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Container\Container.php(627): Illuminate\Container\Container->build('App\\Http\\Contro...', Array)
#2 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(674): Illuminate\Container\Container->make('App\\Http\\Contro...', Array)
#3 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(85): Illuminate\Foundation\Application->make('App\\Http\\Contro...')
#4 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(57): Illuminate\Routing\ControllerDispatcher->makeController('App\\Http\\Contro...')
#5 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\Route.php(201): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'App\\Http\\Contro...', 'login')
#6 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\Route.php(134): Illuminate\Routing\Route->runWithCustomDispatcher(Object(Illuminate\Http\Request))
#7 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\Router.php(704): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#8 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#9 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#11 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#12 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\Router.php(706): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#13 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\Router.php(671): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#14 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\Router.php(631): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#15 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(236): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#16 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#17 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#18 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\VerifyCsrfToken.php(50): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#19 [internal function]: Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#20 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#21 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\View\Middleware\ShareErrorsFromSession.php(49): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#22 [internal function]: Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#23 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#24 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Session\Middleware\StartSession.php(62): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#25 [internal function]: Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#26 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#27 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse.php(37): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#28 [internal function]: Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#29 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#30 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\EncryptCookies.php(59): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#31 [internal function]: Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#32 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#33 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(42): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#34 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#35 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(124): call_user_func_array(Array, Array)
#36 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#37 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#38 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(122): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#39 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(87): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#40 E:\xampp\htdocs\workspaces\blog\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#41 E:\xampp\htdocs\workspaces\blog\server.php(21): require_once('E:\\xampp\\htdocs...')
#42 {main}  
[2015-11-05 03:16:31] local.ERROR: exception 'UnexpectedValueException' with message 'Invalid route action: [App\Http\Controllers\Auth\AuthController]' in E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\Route.php:637
Stack trace:
#0 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\Route.php(107): Illuminate\Routing\Route->parseAction(Array)
#1 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\Router.php(518): Illuminate\Routing\Route->__construct(Array, 'auth/login', Array)
#2 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\Router.php(494): Illuminate\Routing\Router->newRoute(Array, 'auth/login', Array)
#3 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\Router.php(472): Illuminate\Routing\Router->createRoute(Array, 'auth/login', 'Auth\\AuthContro...')
#4 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\Router.php(132): Illuminate\Routing\Router->addRoute(Array, 'auth/login', 'Auth\\AuthContro...')
#5 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(213): Illuminate\Routing\Router->get('auth/login', 'Auth\\AuthContro...')
#6 E:\xampp\htdocs\workspaces\blog\app\Http\routes.php(13): Illuminate\Support\Facades\Facade::__callStatic('get', Array)
#7 E:\xampp\htdocs\workspaces\blog\app\Http\routes.php(13): Illuminate\Support\Facades\Route::get('auth/login', 'Auth\\AuthContro...')
#8 E:\xampp\htdocs\workspaces\blog\app\Providers\RouteServiceProvider.php(41): require('E:\\xampp\\htdocs...')
#9 [internal function]: App\Providers\RouteServiceProvider->App\Providers\{closure}(Object(Illuminate\Routing\Router))
#10 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\Router.php(350): call_user_func(Object(Closure), Object(Illuminate\Routing\Router))
#11 E:\xampp\htdocs\workspaces\blog\app\Providers\RouteServiceProvider.php(42): Illuminate\Routing\Router->group(Array, Object(Closure))
#12 [internal function]: App\Providers\RouteServiceProvider->map(Object(Illuminate\Routing\Router))
#13 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Container\Container.php(503): call_user_func_array(Array, Array)
#14 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Support\Providers\RouteServiceProvider.php(72): Illuminate\Container\Container->call(Array)
#15 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Support\Providers\RouteServiceProvider.php(30): Illuminate\Foundation\Support\Providers\RouteServiceProvider->loadRoutes()
#16 E:\xampp\htdocs\workspaces\blog\app\Providers\RouteServiceProvider.php(29): Illuminate\Foundation\Support\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#17 [internal function]: App\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#18 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Container\Container.php(503): call_user_func_array(Array, Array)
#19 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(734): Illuminate\Container\Container->call(Array)
#20 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(717): Illuminate\Foundation\Application->bootProvider(Object(App\Providers\RouteServiceProvider))
#21 [internal function]: Illuminate\Foundation\Application->Illuminate\Foundation\{closure}(Object(App\Providers\RouteServiceProvider), 17)
#22 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(718): array_walk(Array, Object(Closure))
#23 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\BootProviders.php(17): Illuminate\Foundation\Application->boot()
#24 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(203): Illuminate\Foundation\Bootstrap\BootProviders->bootstrap(Object(Illuminate\Foundation\Application))
#25 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(222): Illuminate\Foundation\Application->bootstrapWith(Array)
#26 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(117): Illuminate\Foundation\Http\Kernel->bootstrap()
#27 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(87): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#28 E:\xampp\htdocs\workspaces\blog\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#29 E:\xampp\htdocs\workspaces\blog\server.php(21): require_once('E:\\xampp\\htdocs...')
#30 {main}  
[2015-11-06 03:02:41] local.ERROR: exception 'UnexpectedValueException' with message 'Invalid route action: [App\Http\Controllers\UserController]' in E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\Route.php:637
Stack trace:
#0 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\Route.php(107): Illuminate\Routing\Route->parseAction(Array)
#1 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\Router.php(518): Illuminate\Routing\Route->__construct(Array, 'login', Array)
#2 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\Router.php(494): Illuminate\Routing\Router->newRoute(Array, 'login', Array)
#3 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\Router.php(472): Illuminate\Routing\Router->createRoute(Array, 'login', 'UserController')
#4 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\Router.php(132): Illuminate\Routing\Router->addRoute(Array, 'login', 'UserController')
#5 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(213): Illuminate\Routing\Router->get('login', 'UserController')
#6 E:\xampp\htdocs\workspaces\blog\app\Http\routes.php(13): Illuminate\Support\Facades\Facade::__callStatic('get', Array)
#7 E:\xampp\htdocs\workspaces\blog\app\Http\routes.php(13): Illuminate\Support\Facades\Route::get('login', 'UserController')
#8 E:\xampp\htdocs\workspaces\blog\app\Providers\RouteServiceProvider.php(41): require('E:\\xampp\\htdocs...')
#9 [internal function]: App\Providers\RouteServiceProvider->App\Providers\{closure}(Object(Illuminate\Routing\Router))
#10 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Routing\Router.php(350): call_user_func(Object(Closure), Object(Illuminate\Routing\Router))
#11 E:\xampp\htdocs\workspaces\blog\app\Providers\RouteServiceProvider.php(42): Illuminate\Routing\Router->group(Array, Object(Closure))
#12 [internal function]: App\Providers\RouteServiceProvider->map(Object(Illuminate\Routing\Router))
#13 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Container\Container.php(503): call_user_func_array(Array, Array)
#14 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Support\Providers\RouteServiceProvider.php(72): Illuminate\Container\Container->call(Array)
#15 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Support\Providers\RouteServiceProvider.php(30): Illuminate\Foundation\Support\Providers\RouteServiceProvider->loadRoutes()
#16 E:\xampp\htdocs\workspaces\blog\app\Providers\RouteServiceProvider.php(29): Illuminate\Foundation\Support\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#17 [internal function]: App\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#18 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Container\Container.php(503): call_user_func_array(Array, Array)
#19 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(734): Illuminate\Container\Container->call(Array)
#20 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(717): Illuminate\Foundation\Application->bootProvider(Object(App\Providers\RouteServiceProvider))
#21 [internal function]: Illuminate\Foundation\Application->Illuminate\Foundation\{closure}(Object(App\Providers\RouteServiceProvider), 17)
#22 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(718): array_walk(Array, Object(Closure))
#23 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\BootProviders.php(17): Illuminate\Foundation\Application->boot()
#24 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(203): Illuminate\Foundation\Bootstrap\BootProviders->bootstrap(Object(Illuminate\Foundation\Application))
#25 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(201): Illuminate\Foundation\Application->bootstrapWith(Array)
#26 E:\xampp\htdocs\workspaces\blog\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(98): Illuminate\Foundation\Console\Kernel->bootstrap()
#27 E:\xampp\htdocs\workspaces\blog\artisan(36): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#28 {main}  
